### DISCUSSION TITLE
Improve table interface

### DISCUSSION THREAD

**Discussion by Olyno** [supabase/supabase#8246] [Feature Requests]
Hi 👋🏻 
I think it could be interesting to have a popover of the column name when hovering it, like Discord does with servers name:

![image](https://user-images.githubusercontent.com/25107942/184378801-e51d18b2-0c3e-4875-abec-ff70c9c77a55.png)
![image](https://user-images.githubusercontent.com/25107942/184379124-1530cc76-1f5c-4b81-8516-bc8d12eeb4f5.png)

*Initial suggestion from **snowshift** on Discord*

**Comment by Hallidayo** [comment]
Hi @Olyno,

I have created a pull request (#8255) for this feature to be added

**Reply by Olyno** [reply]
Small feature, but better accessibility. Thank you for your work!
